easyconfigs/p/pcgr/pcgr-1.0.3.eb

```diff
@@ -0,0 +1,35 @@
+# EasyBuild recipy for PCGR
+easyblock = 'Conda'
+
+name = 'pcgr'
+version = '1.0.3'
+
+homepage = 'https://sigven.github.io/pcgr/index.html'
+description = """The Personal Cancer Genome Reporter (PCGR) is a stand-alone software package for 
+functional annotation and translation of individual tumor genomes for precision cancer medicine."""
+
+toolchain = SYSTEM
+
+builddependencies = [
+   ('Miniconda3', '4.12.0'),
+]
+
+
+dependencies = [
+   ('pcgrr', '1.0.3'),
+]
+
+buildininstalldir = True
+
+requirements = "--file https://raw.githubusercontent.com/sigven/pcgr/v%(version)s/conda/env/lock/pcgr-linux-64.lock --prefix ./pcgr"
+
+modextrapaths = {'PATH': 'pcgr/bin/'}
+
+sanity_check_paths = {
+    'files': ["pcgr/bin/pcgr"],
+    'dirs': [],
+}
+
```

Comment:
```
postinstallcmds = ["cd %(installdir)s/../ && ln -s 1.0.3/pcgr . && ln -s $EBROOTPCGRR/pcgrr ."]
```

I think that covers point 1, to avoid the need for manually putting in the links. Can you check that that does what is expected?

Reply:
I get 
`ERROR: Build of /rds/projects/b/branfosj-bear-apps/devel/2021b/yearwoog-21b/bear-eb/easyconfigs/p/pcgr/pcgr-1.0.3.eb failed (err: 'build failed (first 300 chars): cmd "cd /rds/projects/b/branfosj-bear-apps/devel/2021b/yearwoog-21b/EL8-ice/software/pcgr/1.0.3/../ && ln -s 1.0.3/pcgr . && ln -s $EBROOTPCGRR/pcgrr ." exited with exit code 1 and
output:\nln: failed to create symbolic link \'./pcgr\': File exists\n'`

Reply:
Deleted instaldir and it worked